.gitattributes

```diff
@@ -1,5 +1,11 @@
+# See https://git-scm.com/docs/gitattributes#_pattern_format for more about `.gitattributes`.
+
+# Normalize EOL for all files that Git considers text files
 * text=auto eol=lf
+
+# Mark lock files as generated to avoid diffing
 pnpm-lock.yaml linguist-generated
-package-lock.json linguist-generated
-bun.lockb linguist-generated
-yarn.lock linguist-generated
```

Comment:
There are tests for all package managers that generate these different files, which are committed to the project. Lets keep them as generated.

https://github.com/wxt-dev/wxt/tree/main/packages/wxt/src/core/package-managers/__tests__/fixtures

Reply:
> There are tests for all package managers that generate these different files, which are committed to the project. Lets keep them as generated.
> 
> [`main`/packages/wxt/src/core/package-managers/__tests__/fixtures](https://github.com/wxt-dev/wxt/tree/main/packages/wxt/src/core/package-managers/__tests__/fixtures?rgh-link-date=2025-02-17T20%3A38%3A37Z)

Fixed in [`fc11b3a` (#1447)](https://github.com/wxt-dev/wxt/pull/1447/commits/fc11b3a9b44df3cd5ed543e91d0b2d9918b96fcc)